Makefile

```diff
@@ -0,0 +1,63 @@
+ifndef VERBOSE
+.SILENT:
+endif
+
+ASSET_SECRETS=ASSET-KEYS
+
+.PHONY: development
+development:
+	$(eval export KEY_VAULT=s146d01-kv)
+	$(eval export AZ_SUBSCRIPTION=s146-getintoteachingwebsite-development)
+
+.PHONY: production
+production:
+	$(eval export KEY_VAULT=s146p01-kv)
+	$(eval export AZ_SUBSCRIPTION=s146-getintoteachingwebsite-production)
+
+set-azure-account:
+	echo "Logging on to ${AZ_SUBSCRIPTION}"
+	az account set -s ${AZ_SUBSCRIPTION}
+
+clean:
+	rm -f Google/Code.js Google/.clasp.json Google/creds.json
+	[ ! -f fetch_config.rb ]  \
+	    `rm -f fetch_config.rb` \
+	    || true
+
+install-fetch-config:
+	[ ! -f fetch_config.rb ]  \
+	    && echo "Installing fetch_config.rb" \
+	    && curl -s https://raw.githubusercontent.com/DFE-Digital/bat-platform-building-blocks/master/scripts/fetch_config/fetch_config.rb -o fetch_config.rb \
+	    && chmod +x fetch_config.rb \
+	    || true
+
+npm:
+	$(eval export CLASP_FOLDER=$(shell which clasp))
+	if [ -z "$(CLASP_FOLDER)" ]; then \
+		echo "clasp not found. Installing..."; \
+		npm install @google/clasp -g; \
+		echo installed clasp version: `clasp --version`; \
+	else \
+		echo "'which clasp' found clasp at: $(CLASP_FOLDER)"; \
+		echo using clasp version `clasp --version`; \
+	fi
+
+generate_credentials: npm install-fetch-config set-azure-account
+	./fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT}/${ASSET_SECRETS}  -f shell-env-var -d command -- erb ./Google/creds.json.tmpl > ./Google/creds.json
```

Comment:
`-f` is not used when `-d command` is used

Reply:
It doesn't seem to work if I remove the `-f` or the `-f shell-env-var`